temoa/utilities/db_migration_v3_1_to_v4.py

```
@@ -0,0 +1,290 @@
+    'commodityUStreamProcess': 'commodity_up_stream_process',
+    'SegFrac': 'segment_fraction',
+    'segfrac': 'segment_fraction',
+    'MetaDataReal': 'metadata_real',
+    'MetaData': 'metadata',
+    'Myopicefficiency': 'myopic_efficiency',
+    'DB_MAJOR': 'db_major',
+    'DB_MINOR': 'db_minor',
+}
+CUSTOM_EXACT_ONLY = {'time_season', 'time_season_sequential'}
+CUSTOM_KEYS_SORTED = sorted(
+    [k for k in CUSTOM_MAP.keys() if k not in CUSTOM_EXACT_ONLY], key=lambda k: -len(k)
```

Ruff (0.15.9)

[warning] Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)